docs/manual_deployment.md

```diff
@@ -58,7 +58,8 @@ Additionnally the app should subscribe to the **Pull requests** event.
 
 ## Redirect Github webhooks to your local machine
 
-This can be useful when you are doing development work on Precaution itself and need a simple solution to trace the webhooks sent by GitHub and receive them without exposing your app to the internet.
+This is important when debugging the Precaution workflow and its iteraction with GitHub.
```

Comment:
It's not just important, it's essential/necessary/requires - take your pick, though I'd favour required.

Typo iteraction->interaction